apps/cli/src/index.ts

```diff
@@ -0,0 +1,29 @@
+import "zx/globals";
+import { parseArgs } from "node:util";
+import { CLI_ARGS_OPTIONS, GLOBAL_CONFIG_FILE } from "./constants";
+import { ensureFileInPath, showHelp } from "./utils";
+import { link } from "./link";
+import type { Commands, SupportedCommands } from "./types";
+
+const commands: SupportedCommands = {
+  link,
+};
+
+export const main = async () => {
+  try {
+    await ensureFileInPath(GLOBAL_CONFIG_FILE, JSON.stringify({}));
+    const args = parseArgs({
+      args: process.argv,
+      options: CLI_ARGS_OPTIONS,
+      allowPositionals: true,
+    });
+
+    $.verbose = args.values.debug || false;
```

Comment:
Looks like we do not yet use zx. Let's remove it and see if it really necessary in other commands.

Reply:
We need it in `build` stages. But i am removing it. We can achieve the same using `child_proecess` something like that. `zx` seems to be overkill for just 1-2 commands 👍